src/mumble/VersionCheck.cpp

```diff
@@ -205,3 +215,91 @@ void VersionCheck::fetched(QByteArray a, QUrl url) {
 
 	deleteLater();
 }
+#else // USE_APPIMAGE_UPDATER_BRIDGE
+VersionCheck::VersionCheck(bool autocheck, QObject *p, bool focus) : QObject(p) {
+	Q_UNUSED(focus)
+	m_UpdaterDialog = nullptr;
+	m_Revisioner = new AppImageDeltaRevisioner(/*singleThreaded=*/true,/*parent=*/this);
+	connect(m_Revisioner, &AppImageDeltaRevisioner::updateAvailable,
+		this, &VersionCheck::handleUpdateCheck);
+	connect(m_Revisioner, &AppImageDeltaRevisioner::error,
+		this, &VersionCheck::handleUpdateCheckError);
+
+	if(!autocheck)
+		g.mw->msgBox(QString::fromUtf8("Checking for update."));
+
+	m_Revisioner->checkForUpdate();
+	return;
+}
+
+void VersionCheck::handleUpdateCheck(bool updateAvailable,const QJsonObject &info){
+	(void)info;
+	disconnect(m_Revisioner, &AppImageDeltaRevisioner::updateAvailable,
+		   this, &VersionCheck::handleUpdateCheck);
+	disconnect(m_Revisioner, &AppImageDeltaRevisioner::error,
+		   this, &VersionCheck::handleUpdateCheckError);
+
+	if(!updateAvailable){ // Notify the user they are using the latest version.
+		g.mw->msgBox(QString::fromLatin1("You are currently using the latest version of Mumble AppImage."));
```

Comment:
I'm not sure whether this line should be printed. Afaik the "normal" update check doesn't print anything if no new version is available :thinking:

Reply:
Done.